### PR TITLE
Docs english version

### PR DESCRIPTION
**Pull request by tuanbpl-0873**

### CODE REVIEW

File: document-en.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
+## FramgiaCI Documents
+`Sun*CI` is a CI/CD system developed by Sun* Asterisk , beside the main features like the other CI/CD system, `Sun*CI` provided more features for assisting user in developing projects.Here's some  instruction you better look before intergrate with `Sun*CI`
```
Comment (wataridori) [inline document-en.md]: `Sun* Asterisk` => [Sun Asterisk](https://sun-asterisk.com/)

Comment (wataridori) [inline document-en.md]: `Here's some  instruction` => `Here's some  instructions` 

`you better` => `you'd better`

File: document-en.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+## Permission
+Currently, `Sun*CI` is suporting user login to system using github, gitlab and bitbucket account, after login, by default your permission will be `User`. Your accessibility  to `Sun*CI` base on your permission as show below:
```
Comment (wataridori) [inline document-en.md]: `github, gitlab and bitbucket` => `Github, Gitlab (both https://gitlab.com/ and Sun* Gitlab), Bitbucket`

`after login` => `after logging in`

File: document-en.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+        <td>+</td>
+        <td>-</td>
+        <td>-</td>
+    </tr>
+    
+</table>
+
+   
+## Homepage ( Active Repositories)
+This page display list of projects that you were added by repository's owner and list of repositories that you actived on `Sun*CI` system.
```
Comment (wataridori) [inline document-en.md]: `display` => `displays`

File: document-en.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+        <td>Active project</td>
+        <td>Total number of project actived</td>
+    </tr>
+    <tr>
+        <td>Buid success</td>
+        <td>Total number of build succeeded.</td>
+    </tr>
+    <tr>
+        <td>Build error</td>
+        <td>Total number of error build.Build status will be changed to error when at least one commmand fail.</td>
```
Comment (wataridori) [inline document-en.md]: `.Build` => `. Build`

add space sau dấu `.`

File: document-en.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+
+> You need to be an admin of repository to active project and create hook on github
+> 
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/Selection_013.png)
+
+> 
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/Selection_014.png)
+
+## 4. Overview
+Overview page display sumary information, build statistic of repository.This page show up when you click on a project actived at homepage.
```
Comment (wataridori) [inline document-en.md]: `sumary` => `summary`
`show up` => `is shown up`

File: document-en.md [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/Selection_015.png)
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/Selection_016.png)
+
+>1. The last build information
+>2. Number of builds, success build and error build 
+>3. This chart display errors of 10 recently builds. 
+>4. This chart analyze builds in 10 days. 
+## 5. Build
+By click on each build, detail page will show up, detail page include some detail as below:
```
Comment (wataridori) [inline document-en.md]: `By click` => `By clicking`

`include` => `includes`

`detail` => `details`

File: document-en.md [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+> Logs: Display log result of each command that you defined in config file.
+
+> Coverage: Unit test result
+
+> Violations: Detail information of code convention violated in project.
```
Comment (wataridori) [inline document-en.md]: `Detail information of code convention violated in project` 
=> 
`Detail information of code conventions violated in the project`

File: document-en.md [hunk 8]
```diff
@@ -0,0 +1,244 @@
...
+
+> Metrics: Static analyzer for deverloping language
```
Comment (wataridori) [inline document-en.md]: `deverloping language` => `programming languages`

File: document-en.md [hunk 9]
```diff
@@ -0,0 +1,244 @@
...
+    </tr>
+    <tr>
+        <td>6</td>
+        <td>Scroll to the top of build logs block</td>
+    </tr>
+</table>
+
+
+## 6. Setting & Notification
+You can access `Sun*CI` setting by clicking the `setting tab` on each repository detail page ( You must be an owner or admin of github repository to take this action).
```
Comment (wataridori) [inline document-en.md]: `( You` => `(You` (không có dấu cách sau `(`

File: document-en.md [hunk 10]
```diff
@@ -0,0 +1,244 @@
...
+
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/setting.png)
+
+`Webhook setting` allow you to turn on or off which event github you want to run build.
+
+`Build setting` : You can run build in some specific branch by filling on this field branch name and press Enter.
+
+For example,when you fill `develop` and Enter, `Sun*CI` only will run build with push event to `develop` or someone send a pull request to this branch.
```
Comment (wataridori) [inline document-en.md]: `,when` => `, when`

Comment (wataridori) [reply document-en.md]: `or someone send` => `or when someone sends`

File: document-en.md [hunk 11]
```diff
@@ -0,0 +1,244 @@
...
+
+Beside, `Sun*CI` also bring out a comment on github feature.After every build finish running. `Sun*CI` bot will let you know which line get code convention errors .
```
Comment (wataridori) [inline document-en.md]: `bring` => `brings`

`comment on github` => `Comment-On-Github`

`.After` => `. After`

`finish` => `finishes`

`which line` => `which lines`

File: document-en.md [hunk 12]
```diff
@@ -0,0 +1,244 @@
...
+
+To access this feature, you need to click on `setting` tab and add `Sun*CI` bot to your repository.
+
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/Selection_022.png)
+
+Notification allow user to send message via chatwork,slack or email after every build finish running.
```
Comment (wataridori) [inline document-en.md]: `chatwork,slack or email` => `Chatwork, Slack or Email`

`finish` => `finishes`

File: document-en.md [hunk 13]
```diff
@@ -0,0 +1,244 @@
...
+
+To do this, you need to follow instruction as follows:
```
Comment (wataridori) [inline document-en.md]: `instruction` => `instructions`

File: document-en.md [hunk 14]
```diff
@@ -0,0 +1,244 @@
...
+> Chatwork:
+> 1. Add bot's key
+> 2. Add room id.Remember to add the bot on step 1 to this chatwork room.
```
Comment (wataridori) [inline document-en.md]: `.Remember` => `. Remember`

File: document-en.md [hunk 15]
```diff
@@ -0,0 +1,244 @@
...
+> 3. In case you just want to send message with some specific branch or specific github's event, follow instruction below :
+![available](https://raw.githubusercontent.com/framgiaci/documents/master/images/chatwork.png)
+
+> Slack:
+> 1. Add slack hook ( include `slack webhook` and `slack display name`
+> 2. Add room 
+
+### 6.1. Remote  configuration
+By active this feature, you are using remote configuration yml file instead of `framgia-ci.yml` on your repository.
```
Comment (wataridori) [inline document-en.md]: `active` => `activating`

File: intergration-en.md [hunk 16]
```diff
@@ -0,0 +1,312 @@
+# Integrate FramgiaCI into your Project
+
+## framgia-ci.yml
+
+To intergrate with `Sun*CI`, your project must contain `framgia-ci.yml` file.You can automatically create this file by using [Framgia CI CLI Tool](https://github.com/framgiaci/framgia-ci-cli) or make it by yourself with the file's content below:
```
Comment (wataridori) [inline intergration-en.md]: `.You` => `. You`

File: intergration-en.md [hunk 17]
```diff
@@ -0,0 +1,312 @@
...
+2. `build` <br/>
+    This block contain command for running tools , test command.
+    * `general_test`:name of this test block.It can be: `unit_test`, `feature_test`Etc.  Any keyword you want.
+        + `image: framgiaciteam/laravel-workspace:latest` <br/>
+        Image's name that you use to start a container and run all commands defined inside. In this example, it's `framgiaciteam/laravel-workspace:latest` developed by `Sun*CI` CI/CD team. 
+        You can create your own docker image. Publish on Dockerhub and use it for your project.
+         `image: framgiaciteam/my_docker_image:1.1.0`
+            > By default, `Sun*CI` will add `latest` if you're not specify any tag.
+        + `services` <br/>
+        If your project need to connect to database for running test (unit test, test migrate, test connnect DB etc.), you must specify those services on this block.In this example, Laravel need to connect to `mysql` database, so `services` block will be look like this: <br/>
```
Comment (wataridori) [inline intergration-en.md]: `need` => `needs`

Comment (wataridori) [reply intergration-en.md]: `.In` => `. In`

File: intergration-en.md [hunk 18]
```diff
@@ -0,0 +1,312 @@
...
+        + `prepare`: a set of commands that you want to run in your project. For example : <br/>
+            ```
+                - composer install
+            ```
+
+3. `deploy`
+    This block define your plugins for do some jobs after running test.For example in some cases you want to auto deployment after succeeded build. Currently, `Sun*CI` supporting auto deployment with:
+    * rocketeer
+    * capistrano
+    * docker ( Automatically build project into a docker image, push to dockerhub then pull and update to your server
```
Comment (wataridori) [inline intergration-en.md]: `( Automatically` => `(Automatically`

File: intergration-en.md [hunk 19]
```diff
@@ -0,0 +1,312 @@
...
+        git:
+            folder: .git
+        composer:
+            folder: vendor
+            file: composer.lock
+        nodejs:
+            folder: node_modules
+            file: yarn.lock
+    ```
+    This is Laravel project's example. `vendor` and `node_modules` only change when you add or remove some packages.All those change has been writen in `composer.lock` file and `yarn.lock` file. `Sun*CI` decide to save cache folder base on the changes of these file.
```
Comment (wataridori) [inline intergration-en.md]: `.All` => `. All`

File: intergration-en.md [hunk 20]
```diff
@@ -0,0 +1,312 @@
...
+
+### Active project and sync members
+- To `active` a project,access `Available repository` page -> `Fetch repositories` -> Click on repository need to active -> `Create Hook`.
+
+        > Note: The Project only be actived by admin of repository.
+
+- After actived project. Access `Detail` page -> `setting` and click `Re - SYNC REPO'S MEMBER` incase other members not able to see build's information.
+
+### Receive notification about build's results.
+- By default, when you active a project, `Sun*CI` chatbot will be added to your project. But if you want to get notification from this chatbot, you need to turn on  chatwork notification.Click on `Detail` -> `NOTIFICATION` -> turn on `CHATWORK NOTIFICATION` and add chatwork chatroom ID to `Room Lists`.
```
Comment (wataridori) [inline intergration-en.md]: `.Click` => `. Click`

File: intergration-en.md [hunk 21]
```diff
@@ -0,0 +1,312 @@
...
+- You can also create your own chat bot by add your chatbot key to `Chatwork Bot`
+- Receive build's result via email: add email address to `Email Lists` in `SETTING -> NOTIFICATION`
+- Receive notification via slack: To get notification via slack, you need to turn on slack's notification and add all necessary information to `Slack Notification`.
+
+### Public key
+- `Sun*CI` provide a public key.You can use this key to add to your server ( you must use this key if you want to use auto deployment services). Click on `Detail` -> `SETTING` -> `SECRET`. Copy to use this key.
```
Comment (wataridori) [inline intergration-en.md]: `.You` => `. You`

Comment (wataridori) [reply intergration-en.md]: `( you` => `(you`

File: intergration-en.md [hunk 22]
```diff
@@ -0,0 +1,312 @@
...
+
+## Notes when intergrate with Ruby projects.
+### Gems Cache
+- Ruby projects usually install gems in `/usr/local/bin/gems` and share this folder with other projects in host machine. On the other hand,`Sun*CI` run builds in totally isolated environment, each build in a project run in different environment with the other, that why you can not keep that gem folder in containers.So when you intergrate a Ruby project with `Sun*CI`, please install gem in folder by specify `install path` like:
```
Comment (wataridori) [inline intergration-en.md]: `.So` => `. So`

**Review by daothaison — APPROVED** [review]

**Pull request merged**